packages/fuel-indexer-lib/src/defaults.rs

```diff
@@ -1,55 +1,125 @@
+
+/// Postgres host.
 pub const POSTGRES_HOST: &str = "localhost";
+
+/// Postgres port.
 pub const POSTGRES_PORT: &str = "5432";
+
+/// Postgres password.
 pub const POSTGRES_PASSWORD: &str = "postgres";
 
 pub const INDEX_FAILED_CALLS: usize = 10;
 pub const STOP_IDLE_INDEXERS: bool = false;
 
+/// Max body size for GraphQL API requests.
 pub const MAX_BODY_SIZE: usize = 5242880; // 5MB
 
+/// Size of web-API-to-executor-service channel.
 pub const SERVICE_REQUEST_CHANNEL_SIZE: usize = 100;
+
+/// How long to wait if request to the Fuel GQL client returned no data.
 pub const IDLE_SERVICE_WAIT_SECS: u64 = 3;
 
+/// How many times to retry connecting to the database.
 pub const MAX_DATABASE_CONNECTION_ATTEMPTS: usize = 5;
+
+/// How long to initially wait before retrying to connect to the database.
 pub const INITIAL_RETRY_DELAY_SECS: u64 = 2;
+
+/// Use an embedded PostgresQL database.
+pub const EMBEDDED_DATABASE: bool = false;
+
+/// If using `--stop-idle-indexers`, this is the number of empty blocks after which the executor should stop.
 pub const MAX_EMPTY_BLOCK_REQUESTS: usize = 10;
 
-pub const DELAY_FOR_SERVICE_ERR: u64 = 5;
+/// Amount of time to wait before fetching new blocks, if an executor error occurs.
+pub const DELAY_FOR_SERVICE_ERROR: u64 = 5;
+
+/// Amount of time to wait before fetching new blocks, if the Fuel GQL client returns no data.
 pub const DELAY_FOR_EMPTY_PAGE: u64 = 1;
 
+/// Run migrations on startup.
 pub const RUN_MIGRATIONS: bool = true;
+
+/// Make a metrics endpoint available at `/api/metrics`.
 pub const USE_METRICS: bool = true;
 
+/// Directory at which general Fuel assets and metadata are stored.
 pub const FUEL_HOME_DIR: &str = ".fuel";
+
+/// Directory inside `FUEL_HOME_DIR` at which indexer assets and metadata are stored.
 pub const INDEXER_CONFIG_DIR: &str = "indexer";
 
-pub const ROOT_DIRECTORY_NAME: &str = "fuel-indexer";
+/// Amount of padding for `forc index check` messages.
 pub const MESSAGE_PADDING: usize = 55;
+
+/// Amount of padding for `forc index check` success emojis.
 pub const SUCCESS_EMOJI_PADDING: usize = 3;
+
+/// Amount of padding for `forc index check` error emojis.
 pub const FAIL_EMOJI_PADDING: usize = 6;
+
+/// Amount of padding for `forc index check` headers.
 pub const HEADER_PADDING: usize = 20;
 
+/// Binary name for `forc index` CLI.
 pub const FORC_INDEX: &str = "forc-index";
 
+/// Require users to authenticate for some operations.
 pub const AUTH_ENABLED: bool = false;
-pub const JWT_EXPIRY_SECS: usize = 2592000; // 30 days
 
+/// Amount of time (seconds) before expiring token (if JWT scheme is specified).
+pub const JWT_EXPIRY_SECS: usize = 2592000;
+
+/// Index of account to use for signing.
 pub const ACCOUNT_INDEX: &str = "0";
 
+/// Verbose output.
 pub const VERBOSE_LOGGING: bool = false;
+
+/// Verbose output for database operations.
 pub const VERBOSE_DB_LOGGING: &str = "false";
 
+/// Amount of items to return in a GraphQL page.
```

Comment:
Consider:
> Amount of blocks per response returned by the Fuel node.

Changed "items" to "blocks" since we talk about blocks in the comment for `DELAY_FOR_EMPTY_PAGE`.